tools/cpp/windows_cc_configure.bzl

```diff
@@ -338,17 +395,32 @@ def configure_windows_toolchain(repository_ctx):
     escaped_tmp_dir = escape_string(
         get_env_var(repository_ctx, "TMP", "C:\\Windows\\Temp").replace("\\", "\\\\"),
     )
-    msvc_cl_path = find_msvc_tool(repository_ctx, vc_path, "cl.exe").replace("\\", "/")
+
+    llvm_path = ""
+    if _use_clang_cl(repository_ctx):
+        llvm_path = find_llvm_path(repository_ctx)
+        if not llvm_path:
+            auto_configure_fail("\nUSE_CLANG_CL is set to 1, but Bazel cannot find Clang installation on your system.\n"
+                                + "Please install Clang via http://releases.llvm.org/download.html\n")
+        cl_path = find_llvm_tool(repository_ctx, llvm_path, "clang-cl.exe").replace("\\", "/")
+        link_path = find_llvm_tool(repository_ctx, llvm_path, "lld-link.exe").replace("\\", "/")
+        lib_path = find_llvm_tool(repository_ctx, llvm_path, "llvm-lib.exe").replace("\\", "/")
+    else:
+        cl_path = find_msvc_tool(repository_ctx, vc_path, "cl.exe").replace("\\", "/")
```

Comment:
`find_msvc_tool`'s result is usually used with `.replace("\\", "/")`.

Could you make that the default behavior, built into the function? (Maybe add a `return_mixed_path = True` argument? "Mixed" meaning the same it does for `cygpath -m`.)

Reply:
Read the code more carefully, I think it's OK to always return a mixed path. Thanks!